pkg/controller/user/user_controller_test.go

```diff
@@ -55,7 +62,7 @@ func TestUserController_Empty_User_Spec(t *testing.T) {
 
 	res, err := r.Reconcile(req)
 	if err != nil {
-		t.Fatalf("reconcile returned error: (%v)", err)
+		t.Fatalf("reconcile returned no error: %s", err)
```

Comment:
`%v` might show additional information, when `err` is a struct
```suggestion
		t.Fatalf("reconcile returned no error: %v", err)
```